README.md

```diff
@@ -1,2 +1,27 @@
-# crypto
-Cryptographic primitives used in Polygon Miden rollup
+# Miden Crypto
+This crate contains cryptographic primitives used in Polygon Miden.
+
+## Hash
+[Hash module](./src/hash) provides a set of cryptographic hash functions which are used by Miden VM and Miden Rollup. Currently, these functions are:
```

Comment:
`by Miden VM` should it be `by the Miden VM`?

Reply:
Yep, the latter sounds good. Grammar is not my strong suite :)

Reply:
Ok, then we should probably also say "the Miden Rollup"? and is it Rollup or rollup ?

Reply:
I'd probably say "the Miden rollup" - but I'm not grammar expert :)

Reply:
Done :)